DataFormats/Detectors/Common/include/DetectorsCommonDataFormats/NameConf.h

```diff
@@ -54,12 +54,26 @@ class NameConf : public o2::conf::ConfigurableParamHelper<NameConf>
     return o2::utils::concat_string(prefix, "_", KINE_STRING, ".root");
   }
 
+  // Filename to store geometry file
+  static std::string getGeomFileName(const std::string_view prefix = STANDARDSIMPREFIX)
+  {
+    return o2::utils::concat_string(prefix, "_", GEOM_STRING, ".root");
+  }
+
+  // public standard TTree key (for MC ) -- not a function
+  static constexpr std::string_view MCTTREENAME = "o2sim"; // hardcoded
+
+  // standard name for digitization configuration output
+  static constexpr std::string_view DIGITIZATIONCONFIGFILE = "o2simdigitizerworkflow_configuration.ini";
+
  private:
+  // unmodifiable constants used to construct filenames etc
   static constexpr std::string_view STANDARDSIMPREFIX = "o2sim";
   static constexpr std::string_view HITS_STRING = "Hits";     // hardcoded
   static constexpr std::string_view DIGITS_STRING = "Digits"; // hardcoded
   static constexpr std::string_view GRP_STRING = "grp";       // hardcoded
   static constexpr std::string_view KINE_STRING = "Kine";     // hardcoded
+  static constexpr std::string_view GEOM_STRING = "geometry";
 
   O2ParamDef(NameConf, "NameConf");
```

Comment:
Given that simulation anyway must provide external prefixes, perhaps we indeed don't make this class ConfigurableParam at all, making all names constexpr. Instead, I would make all defaults for the prefix arguments = ``""``, the ``o2sim`` can be added internally, so the non-trivial prefix (or path) would come on top of it.

The reason is that if in the reconstruction I want to take the data (and geometry) from the path ``string XXX``, I don't want to call it as ``NameConf::getGeomFileName( XXX + ' /' + NameConf::STANDARDSIMPREFIX)``,
but simply ``NameConf::getGeomFileName(XXX)``

Reply:
Ok. I'll modify it this way.